messaging/src/main/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessor.java

```diff
@@ -412,7 +412,9 @@ private void statusUpdater(int segmentId, UnaryOperator<TrackerStatus> segmentUp
      *     <li>The {@link ErrorHandler} is defaulted to a {@link PropagatingErrorHandler}.</li>
      *     <li>The {@link MessageMonitor} defaults to a {@link NoOpMessageMonitor}.</li>
      *     <li>The {@code initialSegmentCount} defaults to {@code 16}.</li>
-     *     <li>The {@code initialToken} function defaults to {@link StreamableMessageSource#createTailToken()}.</li>
+     *     <li>The {@code initialToken} function defaults to a {@link org.axonframework.eventhandling.ReplayToken} that starts streaming
```

Comment:
If you copy this piece of text to the `PooledStreamingEventProcessor#builder` method, you can resolve my earlier comment.